example-deployment/k8s/base/rmf-server/rmf-server.yaml

```diff
@@ -107,6 +124,8 @@ spec:
           env:
             - name: RMF_API_SERVER_CONFIG
               value: /rmf-server-config/rmf_server_config.py
+            - name: RMW_IMPLEMENTATION
+              value: rmw_fastrtps_cpp
```

Comment:
I recalled that this was added to workaround minikube misconfiguration, iirc the problem was because minikube isolates the networks so other dds wouldn't work if rmf is ran locally and api-server is ran on the cluster. It only works for fastrtps because it supports shared memory.

I think we should remove this as abusing shared memory features to workaround broken network configuration is wrong. iirc it is not possible to create a configuration that works for other dds without also putting rmf inside the cluster, and even if it is in the cluster, it might still not work because dds requires udp broadcast which might not be supported by minikube.

We can also consider writing some deployment instructions for rmf + kubernetes in the rmf meta repo and link to that here since deploying rmf is out of scope of this repo. It might also make sense to move this example deployment to the meta repo as deployment of rmf + rmf-web has become quite coupled together, making it hard to have separate docs without explaining the other.